scripts/install_maestro

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+#
+# https://github.com/mobile-dev-inc/Maestro have a "curl | bash" style installer but it:
+#
+# * doesn't allow multiple versions to be used at same time in home directory.
+# * modifies you shell startup script(s).
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SOURCE_DIR="$(readlink -f "$(dirname "${BASH_SOURCE[0]}")"/..)"
+. "${SOURCE_DIR}/.common.sh"
+
+VERSION_FILE="${SOURCE_DIR}/.maestro-version"
+VERSION=$(cat "$VERSION_FILE")
+# E.g.      https://github.com/mobile-dev-inc/Maestro/releases/download/cli-1.39.13/maestro.zip
+GITHUB_URL="https://github.com/mobile-dev-inc/Maestro/releases/download/cli-${VERSION}/maestro.zip"
+
+INSTALL_MAESTRO_DIR=~/.install_maestro
+CACHE_DIR="${INSTALL_MAESTRO_DIR}/cache"
+CACHED_VERSION="${CACHE_DIR}/${VERSION}"
+SYMLINKS_DIR="${INSTALL_MAESTRO_DIR}/maestro"
+SYMLINK="${SYMLINKS_DIR}/${VERSION}"
+SYMLINK_TMP="${SYMLINKS_DIR}/${VERSION}.tmp"
+
+log "maestro version: ${VERSION}"
+if [ -L "${SYMLINK}" ] ; then
+    log "already installed."
+    exit 0
+fi
+
+mkdir -p "${SYMLINKS_DIR}"
+rm -fR "${CACHED_VERSION}"
+mkdir -p "${CACHED_VERSION}"
+log "installing from: ${GITHUB_URL}"
+# The standard unzip won't work in a shell pipe, so use the busybox version instead.
+curl -L --silent --show-error "${GITHUB_URL}" | busybox unzip - -d "${CACHED_VERSION}"
```

Comment:
This isn't too pretty, but it works on GitHub Actions (they have busybox by default), and avoids a large temporary file.